airflow-core/src/airflow/ui/public/i18n/locales/de/admin.json

```
@@ -40,7 +40,10 @@
       "standardFields": "Standardfelder"
     },
     "nothingFound": {
-        "title": "Keine Verbindungen gefunden!"
+      "description": "Verbindungen, die durch Secret-Managers oder Umgebungsvariablen definiert sind werden hier nicht gelistet.",
+      "documentationLink": "Mehr Details sind in der Airflow-Dokumentation beschrieben.",
+      "learnMore": "Diese werden zur Laufzeit aufgelöst und sind in dieser Ansicht nicht sichtbar.",
```

```suggestion
      "learnMore": "Diese Verbindungen werden erst eingelesen, wenn ein Task, der sie verwendet ausgeführt wird und sie sind daher in dieser Ansicht nicht sichtbar.",
```

Vorschlag zum Verdeutlichen, für neuere User, die vielleicht nicht wissen was genau mit "Laufzeit" gemeint ist :) 

Und ich hoffe ich habe die Kausalität da richtig interpretiert, ich weiss nicht wie genau Verbindungen resolved werden 😅

---

Super, Danke!